### PR TITLE
WP Export updates

### PR DESCRIPTION
**Pull request by carolinatobon**
R107 + eligibility rule for WPS

@jfix please merge

### CODE REVIEW

File: UserGuide/source/Export_iLibrary_Rules.rst [hunk 1]
```diff
@@ -1,104 +1,104 @@
...
-* Statistical Periodical 
+* Statistical Periodical
 * Book series
-* Country studies series 
+* Country studies series
 * Working Paper Series
 * Working Paper
 
-Kappa items having for publisher: "OECD Publishing / �ditions OCDE" are currently exported with the value 'Paris' as publisher city (oecd:city). 
+Kappa items having for publisher: "OECD Publishing / �ditions OCDE" are currently exported with the value 'Paris' as publisher city (oecd:city).
```
Comment (jfix) [inline UserGuide/source/Export_iLibrary_Rules.rst]: Does this look weird to you too?

File: UserGuide/source/Export_iLibrary_Rules.rst [hunk 2]
```diff
@@ -150,55 +150,57 @@ For Kappa items of catalogue type Working Paper series:
 
 The end year is sent for discontinued serials only, and it corresponds to:
 
-* for Kappa items of catalogue type �Working Paper series�:
+* for Kappa items of catalogue type �Working Paper series�:
```
Comment (jfix) [inline UserGuide/source/Export_iLibrary_Rules.rst]: I would favour straight double quotes as well, but I'm not sure what we've got here.

File: UserGuide/source/Export_iLibrary_Rules.rst [hunk 3]
```diff
@@ -150,55 +150,57 @@ For Kappa items of catalogue type Working Paper series:
...
-The file format (dimensions of the document) is exported to iLibrary for all'Kappa items where a value has been entered in the ``pagesize`` field. 
+The file format (dimensions of the document) is exported to iLibrary for all'Kappa items where a value has been entered in the ``pagesize`` field.
 
 R70 Free Preview full-text item
 -------------------------------
 
-A preview full-text item is exported to iLibrary for all Kappa items which are exported to the iLibrary viewer. The value of this full-text 
-item will be a link to the Keepeek page where the object�s Free Preview is available. 
+A preview full-text item is exported to iLibrary for all Kappa items which are exported to the iLibrary viewer. The value of this full-text
+item will be a link to the Keepeek page where the object�s Free Preview is available.
```
Comment (jfix) [inline UserGuide/source/Export_iLibrary_Rules.rst]: and one more here. an apostrophe.

**Comment by carolinatobon** [comment]
Hi Jakob, the client was playing games. Now I've committed some new stuff. Please review and merge if ok.

**Comment by jfix** [comment]
:lipstick: small typographic changes (apostrophes, quotation marks, etc.)

**Pull request merged**